test/android/app/src/main/java/org/apache/cordova/unittests/StandardActivity.java

```diff
@@ -17,7 +17,7 @@ Licensed to the Apache Software Foundation (ASF) under one
        under the License.
 */
 
-package org.apache.cordova.unittests;
+package org.apache.cordova.unittests2;
```

Comment:
Why is this change required?

Reply:
this PR is about being able to change the package name. I must have changed this for that reason but I don't remember if it is in fact needed so I will revert it